src/net/sf/memoranda/ui/WorkPanel.java

```diff
@@ -162,7 +162,7 @@ public void actionPerformed(ActionEvent e) {
 		notesB.setContentAreaFilled(false);
 		notesB.setFocusPainted(false);
 		notesB.setHorizontalTextPosition(SwingConstants.CENTER);
-		notesB.setText(Local.getString("Notes"));
+		notesB.setText(Local.getString("PSP"));
```

Comment:
As this is a simple switch, let's change the localization files to all have `PSP = PSP` instead of `NOTES = (whatever)`